VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/wr-participant-list-shared.component.ts

```diff
@@ -186,7 +195,9 @@ export abstract class WRParticipantStatusListDirective implements DoCheck {
     }
 
     protected filterObservers(): void {
-        this.observers = this.conference.participants.filter(x => x.hearing_role === HearingRole.OBSERVER);
+        this.observers = this.conference.participants
+            .filter(x => x.hearing_role === HearingRole.OBSERVER || x.hearing_role === HearingRole.QUICK_LINK_OBSERVER)
```

Comment:
should we not use the role property out of the participants, I am suggesting this as the role appears to be an enum that is derived from the UserRole property of video api, where as this hearing role is just a string

Reply:
Essentially they should be the same, but yeah I did start prefering Role over HearingRole for QuickLink users. I will change this.